### DISCUSSION TITLE
The opening of the "The Friends of OpenHTMLtoPDF" organization.

### DISCUSSION THREAD

**Discussion by imario42** [openhtmltopdf/openhtmltopdf#2] [Announcements]
Welcome! 🤗

**Comment by ravasconcelos** [comment]
Hi team, thank you for maintaining this project.

Is there any guideline I can follow to start contributing to this project?

**Reply by imario42** [reply]
Hi!

I don't think that we have something written down, but basically the standard Github workflow: You fork that project, apply your changes on that fork and once they work you create a pull request in github so we can pick them up from there then.

Happy coding! 😀
